pkg/reconcile/reconcilers.go

```diff
@@ -389,13 +389,16 @@ func (r *defaultReconciler) buildNewDeployment(req ReaperRequest) (*appsv1.Deplo
 }
 
 func addAuthEnvVars(deployment *appsv1.Deployment, vars ...*corev1.EnvVar) {
-	initEnvVars := deployment.Spec.Template.Spec.InitContainers[0].Env
+	configInitEnvVars := deployment.Spec.Template.Spec.InitContainers[0].Env
+	schemaInitEnvVars := deployment.Spec.Template.Spec.InitContainers[1].Env
```

Comment:
We don't need to add these env vars to the config init container. Take a a look at the version of this function in `master`. It should only require a slight modification.